.github/workflows/CI.yml

```diff
@@ -16,27 +16,56 @@ jobs:
     strategy:
       matrix:
         java: [11, 17]
-        os: [ubuntu-latest, macos-latest]
 
-    name: Build and Test k-NN Plugin
-    runs-on: ${{ matrix.os }}
+    name: Build and Test k-NN Plugin on Linux
+    runs-on: ubuntu-latest
```

Comment:
Does this mean we dont test on Macos?

Reply:
The test is there. I need to separate it out. Check the next few lines in the same file.

Reply:
https://github.com/opensearch-project/k-NN/blob/3fc9e4852744c53c2740bf2b6d7e480e4479b0fb/.github/workflows/CI.yml#L51